client/pages/home.tsx

```diff
@@ -28,41 +28,7 @@ type Subtask = {
   name: string;
 };
 
-const GOALS: Goal[] = [
-  {
-    goalId: '0',
-    name: 'Complete Chingu Voyage 22',
-    tasks: [
-      {
-        id: '0',
-        name: 'Sprint 1',
-        subtasks: [{ id: '0', name: 'A User Story' }],
-      },
-      { id: '1', name: 'Sprint 2' },
-      { id: '2', name: 'Sprint 3' },
-    ],
-  },
-  {
-    goalId: '1',
-    name: 'Learn GraphQL',
-    tasks: [{ id: '0', name: 'Finish howtographql.com tutorial' }],
-  },
-  {
-    goalId: '2',
-    name: 'Become a React Testing Master',
-    tasks: [
-      { id: '0', name: 'Follow Kent C Dodds' },
-      { id: '1', name: 'Read React-Testing-Library docs' },
-      { id: '2', name: 'Find a Tutorial' },
-    ],
-  },
-  {
-    goalId: '3',
-    name: 'A New Goal',
-  },
-];
-
-const GET_GOALS_QUERY = gql`
+export const GET_GOALS_QUERY = gql`
```

Comment:
Should be moved into `utils/graphql/query` and again reference in `query/index.js` to allow for named import. Subsequently the references to this query should also be updated